openapi.yaml

```diff
@@ -625,65 +645,34 @@ components:
       type: object
       x-examples:
         example-1:
+          account_key: "3fa85f64-5717-4562-b3fc-2c963f66afa6"
           username: maja
-          email: maja@majassen.de
-          password: ahcezie2aiKoon0yaequ3eive7uphie9
           role: user
-          created: '2022-03-07T14:15:44+00:00'
-          changed: '2022-03-07T14:15:44+00:00'
         example-2:
           id: 0
+          account_key: "123e4567-e89b-12d3-a456-426655440000"
           username: string
-          email: user@example.com
-          password: pa$$word
           role: admin
-          created: '2019-08-24T14:15:22Z'
-          changed: '2019-08-24T14:15:22Z'
       x-tira: true
       properties:
+        account_key:
+          type: string
+          format: uuid
+          minLength: 36
+          maxLength: 36
         username:
           type: string
           minLength: 2
           maxLength: 25
```

Comment:
Do we still have a user-specified username after all? My understanding was that only the AccountKey would identify the user.

Reply:
To identify the user to other services, yes. However, you might want to do something like "Hey xyz, welcome back!" in the frontend.

Reply:
I'm going by @ldb's message in the PM channel:
> * Instead of registering with an Email address and a password, a new user will have to provide no personal data
> * Upon registration, a unique Account Key will be generated for the user which acts as some sort of username if you will.

And by the [bouncer wiki](https://github.com/mindtastic/bouncer/wiki/1.-General#high-level-concepts):
> * A user is identified not by email or any kind of user handle, but by a random, 128 Bit ID, the AccountKey 

Which both seem to be at odds with the existence of a username set by the user.

"Hey c75a65c4-458c-4f4e-b83a-c633bfcaa13a, welcome back" :stuck_out_tongue:

Reply:
As I said, this is just for you to personalize the user experience and not to identify the user. If the frontend team decides you don't want or need a username, I can take it out.